doc/user/content/releases/v0.62.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: "Materialize v0.62"
 date: 2023-07-26
-released: false
+released: true
+patch: 2
```

Comment:
```suggestion
patch: 3
```

Reply:
@morsapaes while we cut a v0.62.3, it was not released. Does that matter for the release docs?

Reply:
Sorry, I missed this comment yesterday! The QA team relies on this for CI testing, and AFAIU wants to test against all cut releases, even if these weren't rolled out to users. @philip-stoev can confirm, though!